RInAppMessaging/Classes/Protocols/HttpRequestable.swift

```diff
@@ -8,6 +8,8 @@ internal enum HttpMethod: String {
 
 internal enum RequestError: Error {
     case unknown
+    case missingMetadata
+    case missingParameters
```

Comment:
Do we have tests for the RequestError enum?
If so please add tests for these added errors. If not, it can be done in a different jira/PR.

Reply:
Done. `missingParameters` error is theoretically unreachable but I managed to test it